plugins/updater/guest-js/index.ts

```diff
@@ -55,12 +55,27 @@ class Update extends Resource {
     this.body = metadata.body;
   }
 
+  /** Download the updater package */
+  async download(
+    onEvent?: (progress: DownloadEvent) => void,
+  ): Promise<DownloadedBytes> {
+    const channel = new Channel<DownloadEvent>();
+    if (onEvent !== undefined) {
```

Comment:
let's just check if truthy intstead of not falsy
```suggestion
    if (onEvent) {
```

Reply:
eslint rule prevents me from doing that, do we want to tweak that eslint rule?

> https://typescript-eslint.io/rules/strict-boolean-expressions/

Reply:
I see, just ignore my suggestion